package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "linguist-languages": "^7.5.1",
     "mem": "^8.0.0",
-    "php-parser": "https://github.com/glayzzle/php-parser#425c4dcc9f386596bdcd5b458503b3667a858141"
+    "php-parser": "https://github.com/moddengine/php-parser#c71ee4b5c7bab44dbaf5cb9e9bce7e69b45c23b8"
```

Comment:
Better to wait merge

Reply:
@czosel I have updated this after you merged the changes into the parser